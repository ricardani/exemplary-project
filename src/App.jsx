import React from 'react';

import './app.scss';

const App = () => {
    const helloWorld = 'Hello World!!!!';
    return (
        <div className="app">
            {helloWorld}
        </div>
    );
};

export default App;
