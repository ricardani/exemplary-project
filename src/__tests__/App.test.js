import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

describe('App Component', () => {
    describe('render', () => {
        it('should render correctly', () => {
            expect(shallow(<App />)).toMatchSnapshot();
        });
    });
});
