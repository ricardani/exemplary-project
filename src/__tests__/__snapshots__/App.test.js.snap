// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component render should render correctly 1`] = `
<div
  className="app"
>
  Hello World!!!!
</div>
`;
